src/kernels/ipywidgets-message-coordination/nbExtensionsPathProvider.web.ts

```diff
@@ -0,0 +1,23 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+import { injectable } from 'inversify';
+import { Uri } from 'vscode';
+import { IKernel } from '../types';
+import { INbExtensionsPathProvider } from './types';
+
+@injectable()
+export class NbExtensionsPathProvider implements INbExtensionsPathProvider {
+    getNbExtensionsParentPath(kernel: IKernel): Uri | undefined {
+        switch (kernel.kernelConnectionMetadata.kind) {
+            case 'connectToLiveRemoteKernel':
+            case 'startUsingRemoteKernelSpec': {
+                return Uri.parse(kernel.kernelConnectionMetadata.baseUrl);
```

Comment:
I think it should work, because their url is basically a proxy, hence requests shoudl still flow through.
If not, then there's really nothing much we can do.
I think with Jupyter Hub it will work as jupyter hub is basically just launching a classic Jupyter notbeook application, just as though we launched them manually.